staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -138,6 +139,11 @@ type GenericAPIServer struct {
 	// auditing. The backend is started after the server starts listening.
 	AuditBackend audit.Backend
 
+	// Authorizer determines whether a user is allowed to make a certain request. The Handler does a preliminary
+	// authorization check using the request URI but it may be necessary to make additional checks, such as in
+	// the create-on-update case
```

Comment:
We are changing the permissions necessary for create-on-update; we need a big release note for this. I think it is a good change, it means patch/put never implies create--formerly sometimes it did in a hard to reason about way. But administrators will need to know. Can you add to the feature branch change log file I started so we don't forget?

Reply:
This PR is to master, since it is just affecting create on update, which is broken in master. Should I remake this to the feature branch?